code-challenges.test.js

```diff
@@ -14,21 +14,56 @@
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const fibonacciLength1 = 6
-// Expected output: [1, 1, 2, 3, 5, 8]
+ //const fibonacciLength1 = 6
+// // Expected output: [1, 1, 2, 3, 5, 8]
 
-const fibonacciLength2 = 10
+// describe("myFibonacci", () => {
+//   it("returns an array containing the Fibonacci sequence", () => {})
+//   expect(myFibonacci(fibonacciLength1)).toEqual([1, 1, 2, 3, 5, 8])
+// })
+
+//const fibonacciLength2 = 10
 // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
 // b) Create the function that makes the test pass.
 
-// Pseudo code:
+// const myFibonacci = (fibonacciLength1) => {
+//   const newArray = [1, 1]
+//   let index = 0
+//   for (let i = 1; i < fibonacciLength1 - 1; i ++) {
+//     let sum = newArray [index] + newArray[i]
+//     newArray.push(sum)
+//     index ++
+//   }   
+//   return newArray
+// }
+// console.log(myFibonacci(fibonacciLength1))
```

Comment:
Great work!  Remember you want to make this function dynamic.  Right now it's hardcoded for fibonacciLength1 only.  While the logic is on point, you want to make sure it will work on any input.